src/openvic-simulation/misc/Modifier.cpp

```diff
@@ -67,6 +67,14 @@ bool ModifierValue::has_effect(ModifierEffect const* effect) const {
 	return values.contains(effect);
 }
 
+void ModifierValue::set_effect(ModifierEffect const* effect, fixed_point_t value) {
```

Comment:
Can't you use a reference instead of a pointer here?
That would save the nullcheck.

Reply:
Good point, I'll change that for this and similar functions in `ModifierValue` and `ModifierSum`